README.md

```diff
@@ -133,8 +133,9 @@ This assumes that the pre-production server is otherwise ready.
   -``sudo -u apache /usr/local/bin/wp config set WP_DEBUG false --raw --path=/var/www/html``
   - ``sudo -u apache /usr/local/bin/wp super-cache flush --path=/var/www/html``
  - Make sure the menu background is now black.
- - login to the proxy: ("kielipankki-proxy-prod": ssh cloud-user@195.148.30.210)
- - change the proxy settings in the proxy's Ansible configuration under group-vars/all to point the variable `portal_server_ip` to point to the pre-production server's INTERNAL IP (7/2020 that is: 192.168.1.8). The new server is immediately in use and now considered production.
+ - change the proxy settings in the proxy's Ansible configuration under group-vars/all to point the variable `portal_server_ip` to point to the pre-production server's INTERNAL IP.
+- Update the production proxy configuration: `ansible-playbook -i inventories/production proxy.yml -t apache_config_update` 
```

Comment:
I already hastily clicked approve, but actually these are stuff that's in proxy playbooks, but nothing in the instructions hints towards having to clone another repo. If we want to be "summer ready" in the sense that we can leave the services and their docs to someone not already familiar with Kielipankki, that's something that should be in the instructions.

Altogether it might be best to either
a) keep the proxy related instructions in proxy repo only, and just link to them from other repos (otherwise it's unlikely that we remember to update all of the instructions if we change the proxy provisioning system
or
b) fully automated gay luxury space communism where we have proxy IP updating role available e.g. in ansible-common and each proxied service can take care of their own IPs automatically based on their own inventories and such.

B surely isn't realistic to do right at this instant, but at least I think we should move towards a.